src/test/java/core/basesyntax/strategy/operations/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.strategy.operations;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.Operation;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class SupplyOperationHandlerTest {
+    private static final Operation VALID_OPERATION = Operation.SUPPLY;
+    private static final String VALID_FRUIT = "apple";
+    private static final int VALID_QUANTITY = 20;
+    private static final int SUPPLY_QUANTITY = 10;
+    private static OperationHandler supplyHandler;
+
+    @BeforeAll
+    static void beforeAll() {
+        supplyHandler = new SupplyOperationHandler();
+        Storage.getStorage().put(VALID_FRUIT, VALID_QUANTITY);
+    }
+
+    @AfterAll
```

Comment:
<img width="259" alt="image" src="https://github.com/mate-academy/jv-fruit-shop-tests/assets/72043323/3e90e483-0a2a-450a-b438-35f26cd0c26b">

Since you can have multiple tests (or you will expand your logic so the tests too), you have to use this way of storage cleaning